test/jdk/javax/swing/JColorChooser/8065098/JColorChooserDnDTest.java

```diff
@@ -34,21 +34,17 @@
 public class JColorChooserDnDTest {
 
     public static void main(String[] args) {
-        SwingUtilities.invokeLater(new Runnable() {
-
-            @Override
-            public void run() {
-                JFrame frame = new JFrame();
-                frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-                JPanel panel = new JPanel();
-                JColorChooser colorChooser = new JColorChooser();
-                colorChooser.setDragEnabled(true);
-                panel.setBorder(BorderFactory.createTitledBorder("JColorChoosers"));
-                panel.add(colorChooser);
-                frame.setContentPane(panel);
-                frame.pack();
-                frame.setVisible(true);
-            }
+        SwingUtilities.invokeLater(() -> {
```

Comment:
Copyright year needs to be updated for this file

Reply:
Updated. Thanks.